paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/Action.java

```diff
@@ -18,9 +18,13 @@
 
 package org.apache.paimon.flink.action;
 
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+
 /** Abstract class for Flink actions. */
 public interface Action {
 
     /** The execution method of the action. */
     void run() throws Exception;
+
+    default void build(StreamExecutionEnvironment env) throws Exception {}
```

Comment:
Actions that override this method should be annotated.

Reply:
done.